### ISSUE TITLE
Dockerize the application

### ISSUE THREAD

**Issue by prathameshgaikwad** [prathameshgaikwad/Innings#13]
**Describe the solution you'd like**
I want to Dockerize the application so that it can run consistently in any environment. This involves creating a Dockerfile to containerize the app and a Docker Compose file to manage multi-container setups, if necessary.

**Comment by Essogbe** [comment]
Hi @prathameshgaikwad can you assign me this issue ? I can work on it

**Comment by prathameshgaikwad** [comment]
There you go @Essogbe. The application is relatively simple, however if you face any issues or want any clarification, please don't hesitate to ask!

**Comment by Essogbe** [comment]
Cool. Can you please add ''hacktoberfest'' ''hacktoberfest-accepted'' labels ?

**Comment by prathameshgaikwad** [comment]
Hey @Essogbe, are you still working on this?

**Comment by Essogbe** [comment]
Yes I was a little late. I was able to dockerise the app. I'm working on testing to make sure everything is ok. I'll do the pull request this weekend

**Comment by prathameshgaikwad** [comment]
Okay @Essogbe
